Headers Swedish old translations fixed (#2136)
cerner/terra-framework: packages/terra-notification-dialog/CHANGELOG.md

## Unreleased

* Fixed

>  * Headers Swedish old translations replaced to match the new translations used elsewhere.

Reviewer: ```suggestion
 * Fixed Swedish translations for headers with new translations used elsewhere.
```
